src/parsimmon.js

```diff
@@ -30,8 +30,10 @@
     this._ = action;
   };
 
-  Parsimmon.Parser = Parser;
-
```

Comment:
Would you mind putting that line back, `Parsimmon.Parser = Parser`? And of course removing the accompanying test. I'm going to keep that export after further consideration. But then I'll gladly merge.